docs/paper.md

```diff
@@ -50,57 +50,61 @@ affiliations:
 
-The software repository can be found at [https://github.com/nci/scores/](https://github.com/nci/scores/).
+The `scores` software repository can be found at [https://github.com/nci/scores/](https://github.com/nci/scores/).
 
 \pagebreak
 
 # Statement of Need
 
-The purpose of this software is (a) to mathematically verify and validate models and predictions and (b) to foster research into new scores and metrics. 
+Labelled, n-dimensional data is widely used in many scientific fields. The Earth system science community makes heavy use of physics-based and machine learning models, both to process observations (such as identifying land use from satellite data) and to make predictions about the future (such as forecasting the weather). These models, predictions and forecasts undergo verification and evaluation to assess their correctness. 
+
+The purpose of `scores` is (a) to mathematically verify and validate models and predictions and (b) to foster research into new scores and metrics. 
+
+`scores` handles dimensionality and weighting (e.g. latitude weighting) more effectively than commonly-used data science packages. While there are existing open source Python verification packages for labelled n-dimensional data (see "Related Software Packages" further below), none of these packages offer all of the key benefits of `scores`.
 
 ## Key Benefits of `scores`
 
 In order to meet the needs of researchers and other users, `scores` provides the following key benefits.
 
-**Data Handling**
+### Data Handling
 
-- Works with n-dimensional data (e.g., geospatial, vertical and temporal dimensions) for both point-based and gridded data. `scores` can effectively handle the dimensionality, data size and data structures commonly used for:
+- Works with labelled, n-dimensional data (e.g., geospatial, vertical and temporal dimensions) for both point-based and gridded data. `scores` can effectively handle the dimensionality, data size and data structures commonly used for:
   - gridded Earth system data (e.g. numerical weather prediction models)
   - tabular, point, latitude/longitude or site-based data (e.g. forecasts for specific locations).
 - Handles missing data, masking of data and weighting of results.
 - Supports `xarray` [@Hoyer:2017] datatypes, and works with NetCDF4 [@NetCDF:2024], HDF5 [@HDF5:2020], Zarr [@zarr:2020] and GRIB [@GRIB:2024] data sources among others.
 
-**Usability**
+### Usability
 
 - A companion Jupyter Notebook [@Jupyter:2024] tutorial for each metric and statistical test that demonstrates its use in practice.
 - Novel scores not commonly found elsewhere (e.g. FIRM [@Taggart:2022a], Flip-Flop Index [@Griffiths:2019; @griffiths2021circular]).
+- Commonly-used scores are also provided so that `scores` can be used as a standalone package, and also to meet frequent user requests for their inclusion.
```

Comment:
I think user requests for their inclusion is a stronger reason so may be better to appear before "standalone package" reasoning? e.g. or similar:

```suggestion
- Commonly-used scores are also included, in order to meet frequent user requests to use `scores` as a standalone package.
```

This may not be the actual user request, but if there is evidence that people increasingly want to see "scores" have apis for common metrics. Its an implication that they prefer using scores directly (standalone), than having to use other apis for common metrics.

Reply:
Thanks for the great suggestion. I have accepted Tennessee's very slightly tweaked version of your suggestion.